src/ralph/backends/database/base.py

```diff
@@ -8,10 +8,13 @@ class BaseDatabase(ABC):
     """Base database backend interface."""
 
     name = "base"
+    query_model = None
```

Comment:
```suggestion
    query_model = BaseModel
```
or maybe `BaseQuery`? WDYT?